salt/modules/junos.py

```diff
@@ -432,17 +459,38 @@ def rollback(id=0, **kwargs):
     else:
         op.update(kwargs)
 
+    op['unlock'] = op.get('unlock', False)
+
     try:
         ret['out'] = conn.cu.rollback(id)
     except Exception as exception:
         ret['message'] = 'Rollback failed due to "{0}"'.format(exception)
         ret['out'] = False
+        if op['unlock']:
+            try:
+                conn.cu.unlock()
+            except:
+                pass
```

Comment:
This is pep20 violation which even I am not in favour of
https://www.python.org/dev/peps/pep-0020/

"Errors should never pass silently."